Bugzilla/CGI.pm

```diff
@@ -441,12 +441,16 @@ sub header {
         %headers = ('-type' => shift(@_));
     }
     else {
-        %headers = @_;
+        # we always want headers to be in lower case, to avoid
+        # for instance -Cache_Control vs. -cache_control.
+        my %tmp = @_;
+        %headers = map { lc($_) => $tmp{$_} } keys %tmp;
```

Comment:
I think random is acceptable, but it would be good to call that out in a comment.
I'll do that.